test/blocksToHyperScript.test.ts

```diff
@@ -1,22 +1,27 @@
 /* eslint-disable id-length, max-len */
-const runTests = require('@sanity/block-content-tests')
-const internals = require('../src/internals')
-const blocksToHyperScript = require('../src')
-const getSerializers = require('../src/serializers')
+import runTests from '@sanity/block-content-tests'
+import * as internals from '../src/internals'
+import blocksToHyperScript, {renderNode as h, getImageUrl} from '../src/index'
+import getSerializers from '../src/serializers'
+
+// eslint-disable-next-line no-console
+console.warn = jest.fn() // silences the console.watn calls when running tests
+// REVIEW: could we somehow test when console.warn have been called?
+// it('console.warn have been called', () => {
+//   expect(console.warn).toBeCalledWith(/** */)
+// })
 
-const h = blocksToHyperScript.renderNode
-const getImageUrl = blocksToHyperScript.getImageUrl
 const {defaultSerializers, serializeSpan} = getSerializers(h)
-const normalize = html =>
+const normalize = (html) =>
   html
     .replace(/<br(.*?)\/>/g, '<br$1>')
     .replace(/<img(.*?)\/>/g, '<img$1>')
     .replace(/&quot;/g, '"')
-    .replace(/&#x(\d+);/g, (match, code) => {
+    .replace(/&#x(\d+);/g, (_, code) => {
       return String.fromCharCode(parseInt(code, 16))
     })
-    .replace(/ style="(.*?)"/g, (match, styleProps) => {
-      const style = styleProps.replace(/:(\S)/g, ': $1')
```

Comment:
For some reason, I had to remove this empty space from the `normalize` function for some tests to pass. Any ideas?

Reply:
This has been reverted in [`ba15ec7` (#23)](https://github.com/sanity-io/block-content-to-hyperscript/pull/23/commits/ba15ec7da8238430ce74bed3cdb1f8e5e41dd01b) (I assume adding `tsdx` and removing unnecessary jest config was the key), so no real change has happened, *the test file is exactly the same as before the conversion to TS*! 🎉